apps/ensadmin/src/hooks/useNamespaceId.ts

```diff
@@ -0,0 +1,41 @@
+import { ENSNamespaceIds } from "@ensnode/datasources";
+import { useENSIndexerConfig } from "@ensnode/ensnode-react";
+import { useQuery } from "@tanstack/react-query";
+
+/**
+ * Hook to get the namespace ID from the active ENSNode connection.
+ *
+ * Returns the ENS namespace identifier for the currently configured ENSNode client.
+ * This determines which ENS network (Mainnet, Sepolia, Holesky, etc.) the application
+ * is connected to. Falls back to Mainnet if no configuration is available.
+ *
+ * @returns Query result with namespace ID, loading state, and error handling
+ *
+ * @example
+ * ```typescript
+ * import { useNamespaceId } from "@/hooks/useNamespaceId";
+ * import { ENSNamespaceIds } from "@ensnode/datasources";
+ *
+ * function NetworkIndicator() {
+ *   const { data: namespaceId, isLoading, error } = useNamespaceId();
+ *
+ *   if (isLoading) return <div>Loading network...</div>;
```

Comment:
```suggestion
 *   if (isLoading) return <div>Connecting to ENSNode...</div>;
```

- Namespaces are a different concept than networks.
- My understanding is this `isLoading` state should only happen during the process of connecting to an ENSNode?